rosdep/base.yaml

```diff
@@ -3088,6 +3088,17 @@ libdpkg-dev:
   nixos: [dpkg]
   rhel: [dpkg-devel]
   ubuntu: [libdpkg-dev]
+libdraco:
+  arch: [draco]
+  debian: [libdraco-dev]
+  fedora: [draco-devel]
+  rhel:
+    '*': null
+    '9': [draco-devel]
+  ubuntu:
+    '*': null
+    jammy: [libdraco-dev]
+    kinetic: [libdraco-dev]
```

Comment:
```suggestion
    '*': [libdraco-dev]
    focal: null
```

Reply:
Please also list melodic with null as a package has been released for it, too.

Reply:
The rosdep db is rosdistro-agnostic. This is a list of OS platforms, not rosdistro names.

As long as the key for the rules is different from the ROS package name, this will not cause a problem.